example-streamer-uses/README.md

```diff
@@ -1,66 +1,28 @@
-
-Note that the message we received in the uE_service through the streamer still shows the payload with `@i=3`.
-
-If you again reference the terminal where `mE_client` is running you should see the the response message printed:
-
-> ServiceResponseListener: Received a message: UMessage { attributes: MessageField(Some(UAttributes { id: MessageField(Some(UUID { msb: 112888656101015552, lsb: 9811577761723054400, special_fields: SpecialFields { unknown_fields: UnknownFields { fields: None }, cached_size: CachedSize { size: 0 } } })), type_: UMESSAGE_TYPE_RESPONSE, source: MessageField(Some(UUri { authority_name: "linux", ue_id: 4662, ue_version_major: 1, resource_id: 2198, special_fields: SpecialFields { unknown_fields: UnknownFields { fields: None }, cached_size: CachedSize { size: 0 } } })), sink: MessageField(Some(UUri { authority_name: "me_authority", ue_id: 257, ue_version_major: 1, resource_id: 0, special_fields: SpecialFields { unknown_fields: UnknownFields { fields: None }, cached_size: CachedSize { size: 0 } } })), priority: UPRIORITY_CS4, ttl: None, permission_level: None, commstatus: Some(INTERNAL), reqid: MessageField(Some(UUID { msb: 112888656100425728, lsb: 9811577761723054400, special_fields: SpecialFields { unknown_fields: UnknownFields { fields: None }, cached_size: CachedSize { size: 0 } } })), token: None, traceparent: None, payload_format: UPAYLOAD_FORMAT_UNSPECIFIED, special_fields: SpecialFields { unknown_fields: UnknownFields { fields: None }, cached_size: CachedSize { size: 0 } } })), **payload: Some(b"\nThe response to the request: me_client@i=3")**, special_fields: SpecialFields { unknown_fields: UnknownFields { fields: None }, cached_size: CachedSize { size: 0 } } }
-
-Note that the response also contains `@i=3` showing that we have received a response for the original request containing that in the payload.
-
-Further, you will see printed the deserialized `HelloResponse` ProtoBuf object:
-
-> Here we received response: HelloResponse { message: "The response to the request: me_client@i=3", special_fields: SpecialFields { unknown_fields: UnknownFields { fields: None }, cached_size: CachedSize { size: 0 } } }
-
-### High compute service to mechatronics client
-
-Launch the `mE_service` example in another terminal:
-
-```bash
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:<path/to/vsomeip/lib> cargo run --example mE_service
 ```
-
-Launch the `uE_client` example in another terminal:
-
-```bash
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:<path/to/vsomeip/lib> cargo run --example uE_client -- --endpoint tcp/0.0.0.0:7444
+    cargo run --bin ue_client
+    cargo run --bin me_service
 ```
+### How to run publish subscribe examples on same local machine
 
-We omit a detail explanation of the expected terminal output as it's a mirror of the `Mechatronics client to high compute service` heading above.
+```
```

Comment:
We should have this section look similarly to the RPC section above

* use ## for subheading for PubSub
* use ### for two subsections describing how to run ue_publisher, me_subscriber and then the reverse
* The first section should explain what the intended output should look like and that detail can be omitted from the following section